samples/openvino/model_optimizer/convert_model_pipeline.py

```diff
@@ -0,0 +1,28 @@
+
+import kfp.dsl as dsl
```

Comment:
This file is still in the /samples directory. Could we move it to the /contrib directory?

Reply:
samples are now also moved to contrib